lxd/firewall/drivers/drivers_nftables_templates.go

```diff
@@ -182,7 +182,7 @@ table {{.family}} {{.namespace}} {
 // NDP advertisements that come from the genuine Ethernet MAC address but have a spoofed NDP source MAC/IP adddress
 // we need to use manual header offset extraction. This also drops IPv6 router advertisements from instance.
 // If IP filtering is enabled, this also drops unwanted ethernet frames.
-var nftablesInstanceBridgeFilter = template.Must(template.New("nftablesInstanceBridgeFilter").Parse(`
+var nftablesInstanceBridgeFilter = template.Must(template.New("nftablesInstanceBridgeFilter").Option("missingkey=error").Parse(`
```

Comment:
After working with this some more I'm not sure it does what it says on the tin. For example, the template was still rendering with `<no value>` and the errors I was getting back were from nftables but not from the template engine.

I'll remove it from this PR but would you still like me to add it to a separate PR?

Reply:
I've removed the `missingkey=error` option from this PR. The option doesn't work if using `{{ . }}`, you need to be accessing a field i.e. `{{ .thing }}`. 

It would take quite a rework of the templates to add this so I think we're better off leaving it.